src/terragraph-e2e/e2e/minion/BgpUtils.cpp

```diff
@@ -377,83 +365,45 @@ BgpUtils::runVtyshCmd(const std::string& command) {
   return SysUtils::runCommand(commandVec);
 }
 
-int
-BgpUtils::parseVtyshBgpTable(
-    const std::string& bgpTableOutput,
-    const std::string& firstHeader,
-    std::vector<std::string>& headers,
-    std::vector<std::string>& entries) {
-  // Split output line by line
-  std::vector<folly::StringPiece> bgpTableLines;
-  folly::split('\n', bgpTableOutput, bgpTableLines);
-
-  int rowCount = 0;
-  bool inTable = false;
-
-  for (const auto& line : bgpTableLines) {
-    // Split line by whitespace
-    std::string lineStr = line.toString();
-    std::istringstream lineStrStream(lineStr);
-    std::vector<std::string> lineEntries{
-        std::istream_iterator<std::string>{lineStrStream},
-        std::istream_iterator<std::string>{}};
-
-    if (inTable) {
-      // Table is finished when the next line is empty
-      if (lineStr.empty()) {
-        inTable = false;
-        continue;
-      }
+// Vtysh Bgp Advertised Routes Info List
```

Comment:
delete, function comments only go in header files

Reply:
Addressed the review comments